src/decide.ts

```diff
@@ -51,9 +52,11 @@ const createReasonLogger =
 			case Decision.MATCH:
 				return;
 			case Decision.SIZE_MISMATCH:
-				reason = `its size does not match - (${humanReadableSize(
-					searchee.length,
-				)} -> ${humanReadableSize(candidate.size)})`;
+				reason = `its size does not match - (${
+					Math.abs(sizeDiff) > 10000000 // 10MB is the rounding error
+						? `${humanReadableSize(searchee.length)} -> ${humanReadableSize(candidate.size)}`
+						: `${sizeDiff > 0 ? "+" : ""}${sizeDiff} bytes`
```

Comment:
Do you think adding a `-` instead of nothing would be clearer?

Reply:
That's already covered by the number being negative.